config/routes.rb

```diff
@@ -6,7 +6,7 @@
   root to: 'home#index'
 
   # devise routes
-  devise_for :users, controllers: {
+  devise_for :users, path_prefix: 'my', controllers: {
```

Comment:
Bunu anlamadım? Ne yapıyor `path_prefix`? PR'in diğer yerlerinde de bir karşılığı yok, biraz açıklar mısın işlevini.

Reply:
:path_prefix - Set a prefix to the routes with required route variables.

Kayıtlanma sırasındaki kullanıcı oluşturma ve bir yönetici tarafından elle kullanıcı ekleme sırasında route çakışıyordu.

https://github.com/plataformatec/devise/wiki/How-To:-Manage-users-through-a-CRUD-interface
Burada devise'ın oluşturduğu tüm pathlere bir prefix ekleyerek çakışmanın engelleneceği anlatılmış.

Prefix olarak genellikle id veya başka değişkenler tercih ediliyor. Bu kısımda fikir yürütemedim.